packages/ckeditor5-image/src/imageresize/imageresizeui.js

```diff
@@ -194,6 +288,7 @@ function setOptionOn( value ) {
  *
  * @property {String} resizeOption.name A name of the option used for creating a component.
  * You refer to that name later in the {@link module:image/image~ImageConfig#toolbar}.
- * @property {String} resizeOption.label A label to be displayed with a button.
  * @property {String} resizeOption.value A value of a resize option. `null` value is for resetting an image to its original size.
+ * @property {String} resizeOptions.icon A value of the available icon sizes (`small`, `medium`, `large`, `original`).
```

Comment:
It looks like this property is also optional because it is not required when the dropdown is created. It makes sense only when individual buttons are created (if I understand the code).